src/socket.c

```diff
@@ -475,9 +475,9 @@ int connDisableTcpNoDelay(connection *conn) {
     return anetDisableTcpNoDelay(NULL, conn->fd);
 }
 
-int connKeepAlive(connection *conn, int interval) {
+int connKeepAlive(connection *conn, int interval, const int *probes) {
```

Comment:
why `probes` is passed as a pointer and not by value?

Reply:
[d29f931](https://github.com/valkey-io/valkey/pull/1765/commits/d29f9316019dcbb45e12b6c2cbaabbe2dad60234)

OK, I Changed it. Use "CLI_KEEPALIVE_PROBES".